osutil/kcmdline.go

```diff
@@ -210,6 +211,22 @@ type KernelArgument struct {
 	Value string
 }
 
+// UnmarshalYAML implements the Unmarshaler interface.
+func (ka *KernelArgument) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
we need some tests for this in this package now

Reply:
Tests added